### ISSUE TITLE
typo in create.nc help

### ISSUE THREAD

**Issue by mjwoods** [mjwoods/RNetCDF#1]
In create.nc.Rd:

```tex
\item{share}{The buffer scheme. If \code{FALSE} (default), dataset access is buffered and cached for performance. However, if one or more processes may be reading while another process is writing the dataset, set to \code{FALSE}.}
```
  
Should the last word should be TRUE?

(Reported by Michael Sumner)

**Comment by mjwoods** [comment]
Resolved in b5f00b0c

**Comment by mjwoods** [comment]
Hi @mdsumner , FYI.
